src/lib/validators.ts

```diff
@@ -0,0 +1,83 @@
+import { Address, BuildMetadata, ReleaseMetadata } from 'src/types';
+import { z } from 'zod';
+import { exitWithMessage, strings } from './constants';
+import { getPrivateKey } from './keys';
+import {
```

Comment:
let's organize these in alphabetic order so they're easier to read through later